.devcontainer/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM rust:1-bullseye
```

Comment:
Does this come with rustup?

Reply:
Yep, see [this file](https://github.com/rust-lang/docker-rust/blob/master/1.62.1/bullseye/Dockerfile#L24).